### PR TITLE
Docs - move some files from Scylla Docs to Scylla (in tree) - phase 1

### PR DESCRIPTION
**Pull request by lauranovich**
Moves a small number of files. 

NOTE 

I kept the current splash page - this will change

This build also includes enabling multiversion - I need @dgarcia360 to review as I suspect there may be some issue with it.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This documentation targets developers who are interested in contributing to Scyl
 
 ## Contents
 
+* [Getting Started](/manuals/getting-started/index) - How to Download and Install Scylla
```
Comment (dgarcia360) [inline docs/README.md:7]:
```suggestion
* [Getting Started](manuals/getting-started/index) - How to Download and Install Scylla
```

File: docs/conf.py [hunk 2]
```diff
@@ -150,17 +159,19 @@ class CqlLexer(BashLexer):
...
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master']
+smv_latest_version = 'branch-4.4'
+smv_rename_latest_version = 'stable'
+BRANCHES = []
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Whitelist pattern for remotes (set to None to use local branches only)
-smv_remote_whitelist = r"^origin$"
+smv_remote_whitelist = r"^none$"
```
Comment (dgarcia360) [inline docs/conf.py:174]:
```suggestion
smv_remote_whitelist = r"^origin$"
````

File: docs/conf.py [hunk 3]
```diff
@@ -150,17 +159,19 @@ class CqlLexer(BashLexer):
 # -- Options for redirect extension ---------------------------------------
 
 # Read a YAML dictionary of redirections and generate an HTML file for each
-redirects_file = "_utils/redirections.yaml"
+redirects_file = "../_utils/redirections.yaml"
```
Comment (dgarcia360) [inline docs/conf.py:162]:
````suggestion
redirects_file = "_utils/redirections.yaml"
````

File: docs/conf.py [hunk 4]
```diff
@@ -7,8 +7,17 @@
...
+import sys
+from datetime import date
+from sphinx.util import logging
+import recommonmark
+from recommonmark.transform import AutoStructify
+from sphinx_scylladb_theme.utils import multiversion_regex_builder
+
 logger = logging.getLogger(__name__)
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath('../..'))
```
Comment (dgarcia360) [inline docs/conf.py:19]:
```suggestion
sys.path.insert(0, os.path.abspath('../..'))
```

File: docs/conf.py [hunk 5]
```diff
@@ -7,8 +7,17 @@
...
 from sphinx_scylladb_theme.utils import multiversion_regex_builder
 
+import os
+import sys
+from datetime import date
+from sphinx.util import logging
+import recommonmark
+from recommonmark.transform import AutoStructify
+from sphinx_scylladb_theme.utils import multiversion_regex_builder
+
```
Comment (dgarcia360) [inline docs/conf.py:17]:
```suggestion
````

Comment (dgarcia360) [reply docs/conf.py:17]: Please do not duplicate the imports, they are already in lines 1-8.

File: docs/contents.rst [hunk 6]
```diff
@@ -2,7 +2,7 @@
     :hidden:
     :maxdepth: 2
     
-    index
+    ../manuals/getting-started/index
```
Comment (dgarcia360) [inline docs/contents.rst:5]:
````suggestion
    index
    manuals/getting-started/index
````

File: docs/dev-docs/conf.py [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+#
+source_suffix = ['.rst', '.md']
+autosectionlabel_prefix_document = True
+
+# The encoding of source files.
+#
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'contents'
```
Comment (dgarcia360) [inline docs/dev-docs/conf.py:44]: If you only want to build docs for the manual folder, consider:

```suggestion
master_doc = 'manuals/getting-started/index.rst'
```

File: docs/manuals/getting-started/install_scylla/air-gapped-install.rst [hunk 8]
```diff
@@ -0,0 +1,11 @@
+==============================
+Air-gapped Server Installtion
```
Comment (dgarcia360) [inline docs/manuals/getting-started/install_scylla/air-gapped-install.rst:2]:
```suggestion
Air-gapped Server Installation
````

File: docs/manuals/getting-started/scylla_in_a_shared_environment.rst [hunk 9]
```diff
@@ -0,0 +1,85 @@
+
```
Comment (dgarcia360) [inline docs/manuals/getting-started/scylla_in_a_shared_environment.rst:1]:
```suggestion
````

File: docs/manuals/getting-started/system-configuration.rst [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+   for n in /sys/devices/system/node/node?; do
+       echo $NR_HUGEPAGES > $n/hugepages/hugepages-2048kB/nr_hugepages;
+   done
+
+Huge page configuration is written to :code:`/etc/sysconfig/scylla-server` by the script :code:`/usr/lib/scylla/sysconfig_setup`
+
+
+
+
+
```
Comment (dgarcia360) [inline docs/manuals/getting-started/system-configuration.rst:183]:
```suggestion
````

**Review by dgarcia360 — COMMENTED** [review]
Added comments to include the getting started content in the docs.

Before continuing with the PR, please merge https://github.com/scylladb/scylla/pull/7957/files

After resolving the changes, I'll add more comments after checking if you can enable multi-version with the folder structure proposed.

File: docs/conf.py [hunk 11]
```diff
@@ -7,8 +7,17 @@
 from recommonmark.transform import AutoStructify
 from sphinx_scylladb_theme.utils import multiversion_regex_builder
 
+import os
```
Comment (nyh) [inline docs/conf.py]: Not related to this patch, and we don't need to do it now, but I think eventually we should have a comment in the beginning of this file explaining what it does and who's supposed to run it and why.

File: docs/dev-docs/conf.py [hunk 12]
```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
```
Comment (nyh) [inline docs/dev-docs/conf.py:1]: Why do you add a a dev-docs directory - what do you plan to put there? Is this a mistake?

File: docs/manuals/getting-started/index.rst [hunk 13]
```diff
@@ -0,0 +1,24 @@
+Getting Started
+===============
```
Comment (nyh) [inline docs/manuals/getting-started/index.rst:2]: The new manual/ directory is fine for now to help with the wholesale move, but I ask you to please reconsider later the division of docs/ into subdirectories. I don't know how anybody can know if a document they are looking for is in manuals/ or guides/ or what's the difference between the two, and the difference between those two to design-notes.

File: docs/conf.py [hunk 14]
```diff
@@ -171,7 +171,7 @@ class CqlLexer(BashLexer):
 BRANCHES = []
 smv_branch_whitelist = multiversion_regex_builder(BRANCHES)
 # Whitelist pattern for remotes (set to None to use local branches only)
-smv_remote_whitelist = r"^none$"
+smv_remote_whitelist = r"^origin$"
```
Comment (nyh) [inline docs/conf.py:166]: It's not critical, but we usually don't put in the same series a patch that introduces an error and then fixes it in a following patch. You can merge the two patches (there are several patches like this in this series that can be squashed).
Again, I think this is not critical and would accept such a series, but some other maintainers may be less forgiving ;-)

**Review by nyh — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 14 above)
Comment (lauranovich) [reply docs/conf.py:166]: @nyh - please tell me what needs to be done

File: docs/manuals/getting-started/index.rst (hunk 13 above)
Comment (lauranovich) [reply docs/manuals/getting-started/index.rst:2]: @nyh - in going forward we have two choices
1 - keep the two doc types apart from each other.
2 - integrate them

As I discovered with this pilot PR, I am more in favor of #2. Let me know what you think

File: docs/conf.py (hunk 11 above)
Comment (lauranovich) [reply docs/conf.py]: ah ok - sure we can add comments to the file

File: docs/dev-docs/conf.py (hunk 12 above)
Comment (lauranovich) [reply docs/dev-docs/conf.py:1]: I'll take a look but I suspect it was there before

**Comment by lauranovich** [comment]
@nyh - closing this PR and submitting a new one

**Comment by nyh** [comment]
@lauranovich by the way, if you want to push completely different commits into an existing PR, you just need to "force push" (git push -f) the branch with completely different commits. The benefit of doing this is keeping the old comments.

File: docs/dev-docs/conf.py (hunk 12 above)
Comment (nyh) [reply docs/dev-docs/conf.py:1]: Looking at docs/ today, there is no docs/dev-docs.

File: docs/manuals/getting-started/index.rst (hunk 13 above)
Comment (nyh) [reply docs/manuals/getting-started/index.rst:2]: My point is that maybe (?) it makes sense to "keep the two doc types apart", but the problem is that we don't just have two doc types - we already had (after a former move that I never liked) subdirectories like design-notes/ and guides/ (and also contribute/ and alternator/) and now we're adding a fifth subdirectory manuals/. The problem is not the fact that we have 5 subdirectories - it's that it's not clear to users or to developers what belongs where... What's the difference between "manuals" and "guides", for example? As I said, I don't think we need to do anything about this right now, but it's something we'll need to consider in the future. I think we should not continue to have two subdirectories "manuals" and "guides" - either one will need to be renamed, or these two vague subdirectories can be "opened up" to many subdirectories replacing those two.

File: docs/conf.py (hunk 14 above)
Comment (nyh) [reply docs/conf.py:166]: You can use "git rebase -i" to "squash" together patches that shouldn't remain separate - e.g., one big patch which creates a file and then a patch that fixes an error in the first patch can be "squashed" into one patch.
As I said, I don't consider this critical, but this is the normal way we work in scylla.git so some other maintainers might insist on it.

**Pull request closed without merging**